python/ray/__init__.py

```diff
@@ -24,6 +26,23 @@
     from ctypes import CDLL
     CDLL(so_path, ctypes.RTLD_GLOBAL)
 
+if (platform.system().lower() == "Linux".lower()
+    and "Microsoft".lower() in platform.release().lower()):
+    # WSL's /proc/meminfo has a bug that crashes Python; work around it
+    try:
+        import psutil._pslinux
+    except ImportError:
+        psutil = None
+    psutil_open_binary = None
```

Comment:
I think it would be clearer to set this to `None` in the `except` clause below

Reply:
Ok sure, I can do it here since there's only one line in each block, but the reason I avoid it is that it's a bug-prone pattern in more complex cases.